page-optimize.php

```diff
@@ -54,21 +54,31 @@ function page_optimize_should_concat_css() {
 function page_optimize_js_exclude_list() {
 	$exclude_string = get_option( 'page_optimize-js-exclude' );
 	if ( empty( $exclude_string ) ) {
-		return [];
+		return page_optimize_js_exclude_list_default();
```

Comment:
Mmmm... Not sure I'm a fan. We should let users disable these if they feel like it. We can update the messages in the settings screen to mention defaults, but I'd rather not force defaults when nothing is set. `empty( '' ) === true`

Reply:
This was a dumb mistake on my part. Of course people should be able to have empty exclude lists. I'll fix!